benchmarks/README.md

```diff
@@ -16,16 +16,17 @@ ensure that we were running identical software stacks across the blade machines.
 
 Set these configs on the host machine:
 
-    sudo su
-    echo fs.nr_open = 104857600 >> /etc/sysctl.conf
-    echo fs.file-max = 104857600 >> /etc/sysctl.conf
-    echo vm.max_map_count = 1073741824 >> /etc/sysctl.conf
-    echo kernel.pid_max = 4194300 >> /etc/sysctl.conf
-    echo kernel.threads-max = 4194300 >> /etc/sysctl.conf
-    echo kernel.sched_rt_runtime_us = -1 >> /etc/sysctl.conf
-    exit
```

Comment:
The README for the Tor page has a similar blurb. Do we want to update that one too?

Reply:
Maybe better not to mess with it at this point since this current instructions seemed to work for the "kick the tires" reviewers